src/app/static/src/app/store/adrUpload/actions.ts

```diff
@@ -130,5 +132,23 @@ export const actions: ActionTree<ADRUploadState, RootState> & ADRUploadActions =
             }
         }
         await dispatch("getUploadFiles");
+
+        const uploadComplete = rootState.adrUpload.uploadComplete;
+        if (createRelease && uploadComplete){
```

Comment:
I'm not 100% sure if it is necessary to check if uploadComplete before dispatching createRelease. I wanted to avoid a create release being sent to adr if the proceeding upload had failed. This means the upload error message won't get overwritten with a release error message.

Reply:
That makes sense to me